### PR TITLE
Fixed deprecated method

### LINKED ISSUE #466: Fix deprecated method in ZimManageActivity
`MenuItemCompat.expandActionView(menu.findItem(R.id.action_search));`
This line has a deprecated method - expandActionView().

### PR DESCRIPTION
**Pull request by knightcube**
Fixes #466 

Changes: Fixed Deprecated method

Screenshots/GIF for the change: NA

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -30,3 +30,16 @@ More... http://wiki.kiwix.org/wiki/Communication
...
+### Libraries:
+- [Guava](https://github.com/google/guava)
+- [Dagger](https://github.com/google/dagger)
+- [SquiDB](https://github.com/yahoo/squidb)
+- [Apache](https://github.com/apache/commons-io)
+- [RxJava](https://github.com/ReactiveX/RxAndroid)
+- [ButterKnife](https://github.com/JakeWharton/butterknife)
+- [Retrofit](https://github.com/square/retrofit) + [OkHttp](https://github.com/square/okhttp)
+- Testing:
+  - [JUnit4](https://github.com/junit-team/junit4)
```
Comment (brijeshshah13) [inline README.md]: Please follow proper identation and spacing in all the files and commit the changes again.

**Comment by srv-twry** [comment]
@knightcube I think i am already assigned to #419 and i have already done work on it. I have also added the libraries in the readme. Please remove those changes and focus only on one issue at a time.

**Comment by knightcube** [comment]
I have made the required changes. Please tell me if there are any more changes to be made.

File: app/src/main/java/org/kiwix/kiwixmobile/KiwixMobileActivity.java [hunk 2]
```diff
@@ -290,13 +289,13 @@ public void onActionModeFinished(ActionMode mode) {
...
-            }
-            return true;
-          });
+              .setOnMenuItemClickListener(item -> {
+                tts.readSelection(getCurrentWebView());
+                if (actionMode != null) {
+                  actionMode.finish();
+                }
+                return true;
+              });
```
Comment (srv-twry) [inline app/src/main/java/org/kiwix/kiwixmobile/KiwixMobileActivity.java]: Please don't make unrelated changes.

File: app/src/main/java/org/kiwix/kiwixmobile/KiwixMobileActivity.java [hunk 3]
```diff
@@ -408,8 +407,8 @@ public void onCreate(Bundle savedInstanceState) {
 
       @Override public void onSectionClick(View view, int position) {
         getCurrentWebView().loadUrl("javascript:document.getElementById('"
-            + documentSections.get(position).id
-            + "').scrollIntoView();");
+                + documentSections.get(position).id
+                + "').scrollIntoView();");
```
Comment (srv-twry) [inline app/src/main/java/org/kiwix/kiwixmobile/KiwixMobileActivity.java]: Same

File: app/src/main/java/org/kiwix/kiwixmobile/KiwixMobileActivity.java [hunk 4]
```diff
@@ -531,37 +530,37 @@ private void checkForRateDialog() {
...
+            })
+            .setNegativeButton(negative, (dialog, id) -> {
+              visitCounterPref.setNoThanksState(true);
+            })
+            .setNeutralButton(neutral, (dialog, id) -> {
+              tempVisitCount = 0;
+              visitCounterPref.setCount(tempVisitCount);
+            })
+            .setIcon(ContextCompat.getDrawable(this, R.mipmap.kiwix_icon))
+            .show();
```
Comment (srv-twry) [inline app/src/main/java/org/kiwix/kiwixmobile/KiwixMobileActivity.java]: Same here.

File: app/src/main/java/org/kiwix/kiwixmobile/KiwixMobileActivity.java (hunk 2 above)
Comment (mhutti1) [reply app/src/main/java/org/kiwix/kiwixmobile/KiwixMobileActivity.java]: Use 2 space for tabs, 4 space for line continuations.

**Comment by knightcube** [comment]
i dont know why this is happening...i think i will just have to start from scratch...there is probably something regarding auto indentation in my android studio configuration which i am not sure how to change....i will try to fix the issue once again

File: app/src/main/java/org/kiwix/kiwixmobile/KiwixMobileActivity.java (hunk 2 above)
Comment (knightcube) [reply app/src/main/java/org/kiwix/kiwixmobile/KiwixMobileActivity.java]: Whenever i am making a pull this is how this function looks in android studio

````
private void readAloudSelection(Menu menu) {
        if (menu != null) {
            menu.findItem(R.id.menu_speak_text)
                    .setOnMenuItemClickListener(item -> {
                        tts.readSelection(getCurrentWebView());
                        if (actionMode != null) {
                            actionMode.finish();
                        }
                        return true;
                    });
        }
    }`
```

Comment (knightcube) [reply app/src/main/java/org/kiwix/kiwixmobile/KiwixMobileActivity.java]: And i think this is how it should be - 
```
private void readAloudSelection(Menu menu) {
        if (menu != null) {
            menu.findItem(R.id.menu_speak_text)
                .setOnMenuItemClickListener(item -> {
                      tts.readSelection(getCurrentWebView());
                      if (actionMode != null) {
                          actionMode.finish();
                      }
                      return true;
                  });
        }
 }

```
But i have to manually do this everywhere....is there any way to do it automatically

**Comment by srv-twry** [comment]
@knightcube You may have got a notification from Android studio saying that "this file is indented with 2 spaces" and you might have pressed "indent with 4 spaces".
Probably that is the reason.

**Comment by knightcube** [comment]
i made another pull request here #480....please tell me if it is fine or not
